src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -54,15 +55,19 @@ public boolean isRecognizedFormat(BufferedReader reader) throws IOException {
         return reader.lines().anyMatch(line -> RECOGNIZED_FORMAT_PATTERN.matcher(line).find());
     }
 
+    public String[] getEntries() {
+        return entries;
+    }
+
     @Override
     public ParserResult importDatabase(BufferedReader reader) throws IOException {
         List<BibEntry> bibitems = new ArrayList<>();
 
         // use optional here, so that no exception will be thrown if the file is empty
         String linesAsString = reader.lines().reduce((line, nextline) -> line + "\n" + nextline).orElse("");
 
-        String[] entries = linesAsString.replace("\u2013", "-").replace("\u2014", "--").replace("\u2015", "--")
-                                        .split("ER  -.*\\n");
+        entries = linesAsString.replace("\u2013", "-").replace("\u2014", "--").replace("\u2015", "--")
```

Comment:
This is not what I meant with tests. Have a look at the RISImporterTestFiles class.
It searches for test files in /src/test/resources/org/jabref/logic/importer/fileformat that ar named with RISImporterTest....ris 
and have a corresponding .bib file with the same name 

So you create a minimal RIS file with the ER at the end and a corresponding bib files
This way you can test that the import succeeds without errors